packages/trees/frontend/sirius-components-trees/src/trees/FilterBar.types.ts

```diff
@@ -13,10 +13,23 @@
 
 export interface FilterBarProps {
   onTextChange: React.ChangeEventHandler<HTMLTextAreaElement | HTMLInputElement>;
+  onTextClear: () => void;
   onFilterButtonClick: (enabled: boolean) => void;
   onClose: () => void;
+  text: string | null;
+  option?: FilterBarOption;
```

Comment:
`options : FilterBarOptions | undefined`

Reply:
Ok to pluralize all option references, but the use of optional attribut with ? implies `undefined`. I don't know if it necessary to add `| undefined` ?

Reply:
The goal is to remove the `?`, just to stay consistent with ou code base.

Reply:
Ok